mteb/__init__.py

```diff
@@ -18,4 +18,5 @@
     "MTEB_RETRIEVAL_LAW",
     "MTEB_RETRIEVAL_WITH_INSTRUCTIONS",
     "get_tasks",
+    "TASKS_REGISTRY",
 ]
```

Comment:
Maybe this should be kept alphabetic? (i.e. caps comes before lowercase)

Reply:
fixed